### DISCUSSION TITLE
Examples don't load from the manual page

### DISCUSSION THREAD

**Discussion by nicechocolate** [noah1510/LedController#29] [Q&A] [answered]
I tried loading the examples from the manual page: https://noah1510.github.io/LedController/english/examples.html

All of them give me 404.  I wouldn't mention it, but I need to read them; I don't understand the new library, how to set it up, the V2 example programs compile but don't do anything, and of course I can't upgrade existing sketches.  

Or if you could make the V1 documentation available, I could fix the old programs and then upgrade.

You'll be pleased to hear that (WITH V1.7) I WAS using NodeMCU (ESP8266) processor with no issues, using D0, D1 and D2 for CLK, CS and DIN.  There may be better pins, but NodeMCU has weird names for the pins and I am also using an SPI display on D3,4,5,7,8, with TFT_eSPI library, so I don't have any other spare pins.

**Comment by noah1510** [comment, marked as answer]
> I tried loading the examples from the manual page: https://noah1510.github.io/LedController/english/examples.html

This is a known problem and mentioned in #21. I cannot fix it since it is a problem caused by github pages and I do not know why that problem exists.

If you really need to look at the examples you can [open them on github](https://github.com/noah1510/LedController/tree/master/examples/english) or you can clone the repository and build the documentation yourself.
To build the documentation you need to do the following steps:
* Install git and doygen
* clone the repository using git
* open a terminal in the cloned folder
* simply type `doxygen` into the terminal and press enter
This will generate the english documentation in the folder `doc/html` and you can open `doc/html/index.html` to browse the documentation.

> I don't understand the new library, how to set it up

Did you look at the transition page of the documentation? You can find it under the following link: https://noah1510.github.io/LedController/english/d8/dfe/1_8x_8y_to_2_80_8z.html

**Comment by nicechocolate** [comment]
Thank you for your reply, I really appreciate it. I'm sorry to have bothered you about something you already know. I missed your note on #21 and thought I'd found something that had broken and was worth mentioning.

Re: struggling to understand, it's not a big problem, it was just that I had shelved a working but unfinished sketch that was written with V1.7, and of course, when I came back to it, I'd upgraded to V2 and nothing worked.  I still don't understand, but I'm retired, I have all day to work it out. I read the transition page and it made no sense, I just don't have the C programming background to understand it yet. You cannot fix that problem!

Just delete this, then, unless you think it's worth keeping.

**Reply by noah1510** [reply]
I am working on a more detailed tutorial. Having a more detailed description of what has to be done might help.

I can try to figure out why the code is not working if you are willing to share a snippet of that.
